src/main/java/com/ibm/devops/connect/CloudItemListener.java

```diff
@@ -38,9 +38,9 @@
 
     public CloudItemListener(){
         logPrefix= logPrefix + "CloudItemListener ";
-        if (Jenkins.getInstance().getDescriptorByType(DevOpsGlobalConfiguration.class).isConfigured()) {
-            log.info(logPrefix + "CloudItemListener started...");
-        }
+        //if (Jenkins.getInstance().getDescriptorByType(DevOpsGlobalConfiguration.class).isConfigured()) {
```

Comment:
why was this commented out?  We shouldn't log anything unless there is actually something configured